database/migrations/sql/20180604_06_create_users.sql

```diff
@@ -44,7 +44,7 @@ CREATE TABLE user_account (
   email                            CITEXT UNIQUE,
   phone_number                     VARCHAR(20),
   post_code                        CITEXT,
-  birth_year                       INT,
+  birth_year                       INT DEFAULT NULL,
```

Comment:
We've gone live now, we cannot change any of the migrations any more. Any changes must be done with a new migration.

Reply:
😭